lib/bundler.rb

```diff
@@ -159,12 +159,13 @@ def ruby_scope
     def user_home
       @user_home ||= begin
         home = Bundler.rubygems.user_home
+        user_dirs = home ? %w[.bundle .gem].map {|path| File.join(home, path) } : []
```

Comment:
why is `.gem` called out here? I could set my `GEM_HOME` to something other than `~/.gem`

Reply:
Good question, I didn't think too much on the required folders. I think now we only need the `.bundle`.